examples/machine_learning/client/BUILD

```diff
@@ -20,8 +20,8 @@ cc_binary(
     deps = [
         "//examples/machine_learning/proto:machine_learning_cc_grpc",
         "//examples/utils",
+        "//oak/client:application_client",
         "//oak/client:manager_client",
```

Comment:
Manager_client sounds strange :) It makes sense though

Reply:
Maybe deployment_client or similar?